spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -40,7 +40,12 @@ def compile
       spec = Bundler::StubSpecification.from_stub(stub)
       gem = Tapioca::Gemfile::GemSpec.new(spec)
 
-      Tapioca::Compilers::SymbolTableCompiler.new.compile(gem)
+      rbi = Tapioca::Compilers::SymbolTableCompiler.new.compile(gem)
+      rbi.nest_singleton_methods!
+      rbi.nest_non_public_methods!
+      rbi.group_nodes!
+      rbi.sort_nodes!
+      rbi.string
```

Comment:
Ditto about moving this here: https://github.com/Shopify/tapioca/blob/main/lib/tapioca/rbi_ext/model.rb.